widget/models.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import unicode_literals
+
+import logging
+
+import django.db.models
+import django.core.cache
+import django.core.urlresolvers
+
+import shortuuidfield
+
+
+logger = logging.getLogger(__name__)
+
+
+class WidgetSubmission(django.db.models.Model):
+    contact_email = django.db.models.EmailField()
+    contact_name = django.db.models.CharField(max_length=200)
+    company_name = django.db.models.CharField(max_length=100, unique=True, db_index=True)
```

Comment:
@danaspiegel Why we don't need it? Don't we need it for tracking? Do you mean that company_home_url is sufficient?